modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -254,6 +256,9 @@ public SpringCodegen() {
                 "Use the `ResponseEntity` type to wrap return values of generated API methods. "
                     + "If disabled, method are annotated using a `@ResponseStatus` annotation, which has the status of the first response declared in the Api definition",
                 useResponseEntity));
+        cliOptions.add(CliOption.newBoolean(USE_ENUM_CASE_INSENSITIVE,
+                "Use equalsIgnoreCase when String for enum comparison",
```

Comment:
minor - wrap equalsIgnoreCase with ` so that it will generate `equalsIgnoreCase`

Reply:
done